### PR TITLE
Website: Update redirects paths

### PR DESCRIPTION
**Pull request by TuvalSimha**

### CODE REVIEW

File: website/next.config.mjs [hunk 1]
```diff
@@ -2,7 +2,29 @@ import { withGuildDocs } from '@theguild/components/next.config';
...
+    Object.entries({
+      '/docs/basics/mutations.html': '/docs/data/mutations',
+      '/docs/1.0/basics/setup': '/v1',
+      '/docs/basics/setup.html': '/docs/get-started',
+      '/docs/1.0/features/cache-updates': 'v1/basics/setup',
+      '/docs/1.0/basics/caching': '/v1/basics/queries',
+      '/docs/features/cache-updates': '/docs/caching/configuration',
+      '/docs/Roboto-Regular.ttf': '/docs',
+      '/docs/basics/queries.html': '/docs/data/queries',
+      '/docs/1.0/basics/network-layer': '/v1/basics/network-layer',
```
Comment (B2o5T) [inline website/next.config.mjs:14]: Your should use a dynamic pattern here

Comment (TuvalSimha) [reply website/next.config.mjs]: Found a way to do it :) what do you think?

File: website/next.config.mjs [hunk 2]
```diff
@@ -3,27 +3,19 @@ import { withGuildDocs } from '@theguild/components/next.config';
...
+      '/docs/2.0/:slug*': '/v2/:slug*', // Redirect /docs/2.0/... to /v2/...
+      '/docs/2.0/caching': '/v2/caching/configuration',
+      '/docs/2.0/data': '/v2/data/queries',
+      '/docs/2.0/local-state': '/v2/local-state/management',
+      '/docs/2.0/development-and-testing': '/v2/development-and-testing/using-typescript',
+      '/docs/2.0/performance': '/v2/performance/improving-performance',
+      '/docs/2.0/recipes': '/v2/recipes/simple-example',
+      '/docs/2.0/migration': '/v2/migration',
+      '/docs/1.0/:slug*': '/v1/:slug*', // Redirect /docs/1.0/... to /v1/...
+      '/docs/1.0/basics': '/v1/basics/setup',
```
Comment (B2o5T) [inline website/next.config.mjs:15]: All below can be removed?

File: website/next.config.mjs [hunk 3]
```diff
@@ -3,27 +3,19 @@ import { withGuildDocs } from '@theguild/components/next.config';
...
-      '/docs/2.0/caching/field-behavior': '/v2/caching/field-behavior',
-      '/docs/1.0/basics/mutations': '/v1/basics/mutations',
-      '/docs/2.0/data/queries': '/v2/data/queries',
-      '/docs/1.0/features/static-typing': '/v1/features/static-typing',
-      '/docs/1.0/guides/testing': '/v1/guides/testing',
-      '/docs/1.0/recipes/server-side-rendering': '/v1/recipes/server-side-rendering',
-      '/docs/guides/testing.html': '/docs',
-      '/docs/1.0/features/caching': '/v1/features/caching',
+      '/docs/2.0/:slug*': '/v2/:slug*', // Redirect /docs/2.0/... to /v2/...
+      '/docs/2.0/caching': '/v2/caching/configuration',
```
Comment (B2o5T) [inline website/next.config.mjs:7]: All below can be removed?

**Pull request merged**